modules/visual_script/visual_script_builtin_funcs.cpp

```diff
@@ -1105,6 +1116,16 @@ void VisualScriptBuiltinFunc::exec_func(BuiltinFunc p_func,const Variant** p_inp
 			*r_return=ret;
 
 		} break;
+		case VisualScriptBuiltinFunc::COLORN: {
+
+			VALIDATE_ARG_NUM(1);
+
+			Color color = Color::named(*p_inputs[0]);
+			color.a=*p_inputs[1];
+			
```

Comment:
Well, @akien-mga would fix them anyway :stuck_out_tongue: 

Anyway, that's done like so everywhere in the code -- spaces around declarations of new variables only, so decided to follow that one for the case... :smile: